### PR TITLE
doc: opt animator doc

### PR DESCRIPTION
**Pull request by luzhuang**
### Please check if the PR fulfills these requirements

- [x] The commit message follows our [guidelines](https://github.com/galacean/engine/blob/main/.github/COMMIT_MESSAGE_CONVENTION.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Summary by CodeRabbit

- **New Features**
	- Added new documentation files for animation blending, controlling animations by user input, animation transitions, playing animations in models, reusing animations, material animations, and text animations.
- **Improvements**
	- Enhanced clarity and consistency in terminology across multiple documents, especially for Chinese-speaking users.
	- Expanded guidance on creating and exporting animation clips in Unity.
	- Improved structure and organization of existing documentation to facilitate user understanding.
- **Bug Fixes**
	- Minor grammatical adjustments and formatting changes for better readability.

### CODE REVIEW

File: docs/zh/animation/clip.mdx [hunk 1]
```diff
@@ -76,14 +79,14 @@ label: Animation
 
 ### 帧动画示例
```
Comment (eyworldwide) [inline docs/zh/animation/clip.mdx:80]: 各个示例建议放到单独文档里，清晰一点，这篇文档看起来太长了

File: docs/zh/animation/layer.mdx [hunk 2]
```diff
@@ -34,18 +36,23 @@ label: Animation
 
 ## 编辑器使用
 ### 叠加模式
```
Comment (eyworldwide) [inline docs/zh/animation/layer.mdx:38]: 这段没必要再单独列一个“编辑器使用”，感觉有点重复，直接合并到上面章节的“叠加模式”、“覆盖模式”、“混合权重” 中就可以。讲完概念直接讲使用，就说明白了。

File: docs/zh/animation/animator.mdx [hunk 3]
```diff
@@ -16,30 +16,30 @@ label: Animation
...
+模型的 `动画控制组件` 在glTF实例的根实体上，也就是编辑器的模型实体下的第一个子 `实体` 上。
 
-如果模型内有动画，会为你自动绑定一个只读的[动画控制器](/docs/animation/animatorController/)。
+如果模型内有动画，会为你自动绑定一个只读的 `动画控制器`。
 </Callout>
 
 ![alt text](https://mdn.alipayobjects.com/huamei_3zduhr/afts/img/A*WkafRagPFo8AAAAAAAAAAAAADsJ_AQ/original)
 
-如果没有动画控制组件（[Animator](/apis/core/#Animator)）可以按下图方式创建
+如果没有 `动画控制组件` 可以按下图方式创建
```
[the remark this answers is not in the document]
Comment (luzhuang) [reply docs/zh/animation/animator.mdx:33]: done

File: docs/zh/animation/animator.mdx [hunk 4]
```diff
@@ -49,7 +49,7 @@ label: Animation
 
 ### 播放动画
 
-在加载 GLTF 模型后引擎会自动为模型添加一个 Animator 组件，并将模型中的动画片段加入其中。可以直接在模型的根实体上获取 Animator 组件，并播放指定动画。
+在加载 GLTF 模型后引擎会自动为模型添加一个 `动画控制组件` ，并将模型中的 `动画片段` 加入其中。可以直接在模型的根 `实体` 上获取 `动画控制组件` ，并播放指定动画。
```
[the remark this answers is not in the document]
Comment (luzhuang) [reply docs/zh/animation/animator.mdx:52]: 已删除

File: docs/zh/animation/animatorController.mdx [hunk 5]
```diff
@@ -4,29 +4,41 @@ title: 动画控制器
...
+`动画层` 用于在一个 `动画控制器` 中组织和管理 `动画状态机` ，每个 `动画层` 都有其自己的 `动画状态机`。
+在 `动画控制组件` 运行时所有的 `动画层` 同时运行，通过 `混合权重` 及 `混合模式` 控制各个 `动画层` 对整体动画的影响。关于 `动画层` 的使用请参考[动画层](/docs/animation/layer/)文档。
+
+#### 动画参数
+`动画参数` 用于控制 `动画状态机` 的 `动画状态` 切换，通过设置 `动画参数` 的值满足 `动画过渡` 的条件时，`动画过渡` 会触发，从而切换到目标 `动画状态`。关于 `动画参数` 的使用请参考[动画状态机](/docs/animation/state-machine/)文档。
 
 ## 编辑器使用
 
-通过动画控制器的编辑器，用户可以在其中组织[动画片段](/docs/animation/clip)的播放逻辑
+通过 `动画控制器` 的编辑器，用户可以在其中组织 `动画片段`（[详细介绍](/docs/animation/clip)）的播放逻辑
```
[the remark this answers is not in the document]
Comment (luzhuang) [reply docs/zh/animation/animatorController.mdx:22]: 优化为小图

File: docs/zh/animation/clip.mdx [hunk 6]
```diff
@@ -5,15 +5,15 @@ type: 动画
...
 
-**动画片段**是 Galacean 动画系统的核心元素之一，Galacean 支持导入外部设计软件设计的模型动画，设计师输出的带动画的模型中的每个动画在 Galacean 中会被解析成一个个的**动画片段**资产，我们也可以通过动画片段编辑器创作额外的动画。
+`动画片段`（[API](/apis/core/#AnimationClip)）是 Galacean 动画系统的核心元素之一，Galacean 支持导入外部设计软件设计的模型动画，设计师输出的带动画的模型中的每个动画在 Galacean 中会被解析成一个个的 `动画片段` 资产，我们也可以通过动画片段编辑器创作额外的动画。
 
-有两种常用的方式得到动画片段
+有两种常用的方式得到 `动画片段`
 
 1. 导入用第三方工具（例如 Autodesk® 3ds Max®， Autodesk® Maya®， Blender）创建的带动画的模型，详见[美术制作动画片段](/docs/animation/clip-for-artist)
 
 ![1](https://mdn.alipayobjects.com/huamei_3zduhr/afts/img/A*Qc8sQ6iJd8IAAAAAAAAAAAAADsJ_AQ/original)
```
[the remark this answers is not in the document]
Comment (luzhuang) [reply docs/zh/animation/clip.mdx:14]: done

File: docs/zh/animation/clip.mdx (hunk 1 above)
Comment (luzhuang) [reply docs/zh/animation/clip.mdx:80]: done

File: docs/zh/animation/layer.mdx (hunk 2 above)
Comment (luzhuang) [reply docs/zh/animation/layer.mdx:38]: done

**Review by eyworldwide — APPROVED** [review]

**Pull request merged**